content/zh/2020-12-13-skywalking-alarm/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: skywalking报警发送到钉钉群
+date: 2020-12-13
+author: tristan-tsl
+description: 本文将详细介绍与以往做法不同的是如何配置告警发送到钉钉群
+---
+
+
+
+这篇文章暂时不讲告警策略, 直接将默认情况下的快速尝试看到tracing数据监控过程以及告警目标以及钉钉上的告警效果
+
+skywalking内置了很多默认的告警策略, 然后根据告警策略生成告警目标, 我们可以很容易的在界面上看到
+
+![image-20201213163408221](index.assets/image-20201213163408221.png)
+
+当我们想去让这些告警目标通知到我们时, 根据我们的以往的经验(alertmanager), 我们会去找一个skywalking-webhook-dingtalk这样的一个服务来转发, 但是实际上, skywalking目前版本已经自带了, 只需要简单配置一下即可
+
+
+
+如果你是默认的配置文件, 直接执行一下命令即可(或者手动修改configs/alarm-settings.yml)
+
+当然在这之前你需要去钉钉里面创建机器人, 勾选加签
+
+![image-20201213164116760](index.assets/image-20201213164116760.png)
```

Comment:
You don't need the sub folder, according to the doc, https://github.com/apache/skywalking-website#images

Reply:
yes